core/statedisagg.gms

```diff
@@ -111,13 +111,11 @@ $gdxin %gdxdir%shares_usatrd.gdx
 $load usatrd_shr
 $gdxin
 
-* For years not included in USA Trade Online shares, use most recent
-* shares. Earliest year for exports is: 2002. Earliest year for imports
-* is: 2008. For agricultural data, earliest year is 2000 for exports.
+* Relate years of data to available trade shares
 
 usatrd_shr(yr,r,g,'exports')$(yr.val<2002 and not sameas(g,'agr')) = usatrd_shr('2002',r,g,'exports');
 usatrd_shr(yr,r,g,'exports')$(yr.val<2000 and sameas(g,'agr')) = usatrd_shr('2000',r,g,'exports');
-usatrd_shr(yr,r,g,'imports')$(yr.val<2008) = usatrd_shr('2008',r,g,'imports');
+usatrd_shr(yr,r,g,'imports')$(yr.val<2002) = usatrd_shr('2002',r,g,'imports');

```

Comment:
Curious.  How did this happen -- was 2008 a typo? @andrewwschreiber

Reply:
Not in the initial build (v0) -- all we had was import data starting in 2008. I noticed that newer data pulls had backfilled some of the information in windc_build.gdx.